src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1103,6 +1103,12 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
         $translator = $container->findDefinition('translator.default');
         $translator->addMethodCall('setFallbackLocales', [$config['fallbacks']]);
 
+        if (\array_key_exists('cache_dir', $config)) {
```

Comment:
isset() would be more common, and accurate here

Reply:
Isn't this always `true`?

Reply:
I have remove the condition and change the catch value. if the value is empty the cache_dir is null.